experiences/components/chart/view/ChartViewOverviewUber.vue

```diff
@@ -38,35 +38,35 @@
         <VRow>
           <VCol cols="12" md="4">
             <div id="service-chart">
-              <strong>Service used</strong>
+              <strong v-t="k('Service used')" />
               <a class="reset" style="display: none">reset</a>
               <p class="filters">
                 <span>
-                  Current filter:
+                  <span v-t="'Current filter'" />
                   <span class="filter" />
                 </span>
               </p>
             </div>
           </VCol>
           <VCol cols="12" md="4">
             <div id="week-chart">
-              <strong>Day of week</strong>
+              <strong v-t="k('Day of week')" />
               <a class="reset" style="display: none">reset</a>
```

Comment:
We have a translation key for reset too: 
`<a class="reset" style="display: none" v-t="'reset'"></a>`